contracts/.solhint.json

```diff
@@ -3,11 +3,10 @@
   "plugins": ["prettier"],
```

Comment:
(Arbitrary comment location)

 First pass through testing on Optimism, Setup + Send worked flawlessly! 🎉 

First issue I hit was related to withdrawal, which failed with the following JSON-RPC error:

<img width="697" alt="Screen Shot 2022-03-17 at 11 16 19 PM" src="https://user-images.githubusercontent.com/1479802/158930950-d09dddd0-1e27-4eca-be3a-7d5b999fc470.png">

Reply:
I saw this the first time I tried, added a bunch of `console.log` statements for fancy debugging, then of course the second time it worked. I then changing my RPC to my own Alchemy one instead of the default https://mainnet.optimism.io, tried another send > withdraw, and that also worked. So I wonder if this was some temporary RPC flakiness? I sent 0.01 ETH which should be plenty of margin for Optimism gas prices. Can you try again and let me know what happens?

------------------------

Also, because we can't perfectly sweep ETH and leave a few cents of dust, after refreshing the page the table still thinks funds are not withdrawn because balance is non-zero. So we'll have to change how the "withdrawn" status is determined for Optimism/Arbitrum. Am I correct that token withdrawals will not leave dust, so this dust issue only applies to ETH withdrawals?

Reply:
Re (1), I have now seen it again. I just pushed 70d8436 which I _think_ may fix it, but need to test a few more times. If that doesn't work, we probably just need to add some padding to the txCost estimate. Relevant Optimism docs: [1](https://community.optimism.io/docs/developers/build/transaction-fees/), [2](https://community.optimism.io/docs/developers/build/transaction-fees/#insufficient-funds), [3](https://community.optimism.io/docs/guides/wallet-dev/#sending-max-eth)

Reply:
Still seeing this error attempting a withdrawal on Optimism:

<img width="759" alt="Screen Shot 2022-03-19 at 9 05 05 AM" src="https://user-images.githubusercontent.com/1479802/159122250-593888c0-053d-43b2-a3dc-b3cf89fde682.png">

Reply:
This should now be resolved with 4788b35

Reply:
Have not seen this on either network in my testing 👍